lib/matplotlib/offsetbox.py

```diff
@@ -35,7 +35,6 @@
 from matplotlib.patches import bbox_artist as mbbox_artist
 from matplotlib.text import _AnnotationBase
 
-
```

Comment:
not to start an endless discussion about whitespace but I'd put two blank lines here :p